### PR TITLE
Bug fixes and optimisation

### PR DESCRIPTION
**Pull request by hoptond**
- Fixes white space appearing on the bottom of the page at larger resolutions.

- Website now scales better at larger resolution.

- Fixes the showcase section background appearing as white on IE

- Buttons now properly centered on all screensizes in the showcase viewer

- Various minor visual fixes

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -103,10 +105,11 @@ header p {
 }
 
 .showcase {
-    margin: 0;
+    margin: 0 0 73.8px 0;
```
Comment (mporam) [inline style.css]: pixels dont have decimals. round this to 74, as thats what the browser does anyway

File: style.css [hunk 2]
```diff
@@ -296,3 +302,47 @@ ul li a{
...
     }
 }
+@media only screen and (min-width: 2400px) {
+    .navwrapper,
+    .headerwrapper,
+    .showcasewrapper,
+    .contactwrapper {
+        max-width: 80%;
+    }
+    .navitem a{
```
Comment (mporam) [inline style.css]: formatting

**Review by mporam — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (hoptond) [reply style.css]: resolved in ee4eddb

File: style.css (hunk 2 above)
Comment (hoptond) [reply style.css]: resolved in ee4eddb

File: style.css [hunk 3]
```diff
@@ -103,10 +105,11 @@ header p {
 }
 
 .showcase {
-    margin: 0;
+    margin: 0 0 74px 0;
     padding: 0;
     background-color: #a8b5ab;
     width: 100%;
+    height:auto;
```
Comment (mporam) [inline style.css]: formatting

**Review by mporam — CHANGES_REQUESTED** [review]

File: style.css (hunk 3 above)
Comment (hoptond) [reply style.css]: fixed in e9c927a

**Review by mporam — APPROVED** [review]

**Pull request merged**